docs/en/developers/03_Backend/05_Extensions.md

```diff
@@ -296,9 +336,9 @@ In addition, you will have a number of methods directly inherited from `Minz_Ext
 
 - The "getters" first: most are explicit enough not to detail them here - `getName()`, `getEntrypoint()`, `getPath()` (allows you to retrieve the path to your extension), `getAuthor()`, `getDescription()`, `getVersion()`, `getType()`.
 - `getFileUrl($filename, $type)` will return the URL to a file in the `static` directory. The first parameter is the name of the file (without `static /`), the second is the type of file to be used (`css` or` js`).
-- `registerController($base_name)` will tell Minz to take into account the given controller in the routing system. The controller must be located in your `Controllers` directory, the name of the file must be` <base_name>Controller.php` and the name of the `FreshExtension_<base_name>_Controller` class.
+- `registerController($base_name)` will tell Minz to take into account the given controller in the routing system. The controller must be located in your `controllers` directory, the name of the file must be` <base_name>Controller.php` and the name of the `FreshExtension_<base_name>_Controller` class.
```

Comment:
@Alkarex So you'd like me to remove this diff from the PR?

Reply:
Please wait a bit till I find out how it should be :-)

Reply:
Yes, as far as I can see, let's go uppercase: https://github.com/FreshRSS/FreshRSS/pull/1729

Reply:
It depends if you want to document the current state (lowercase) or the upcoming change (uppercase). The change to uppercase will introduce a BC break (but lets discuss that in a #1729).

Reply:
The documentation follows the code, i.e. the documentation in master is for master, and in dev for dev. So it should be for the upcoming change :-)